src/imagery/i.sentinel/i.sentinel.import/i.sentinel.import.html

```diff
@@ -109,12 +109,22 @@ <h2>DESCRIPTION</h2>
 </tbody>
 </table>
 
+
 <h2>NOTES</h2>
 
 <p>
-If <b>-c</b> flag is given, than also cloud mask file is imported as
-vector map if available. The name of created vector map is determined from
-input Sentinel product.
+<b>Importing cloud and cloud shadow masks:</b>
```

Comment:
```suggestion
<h3>Importing cloud and cloud shadow masks</h3>
```

(like this it becomes part of the table of content)